frontend/public/components/app.jsx

```diff
@@ -207,10 +207,10 @@ class App extends React.PureComponent {
   }
 }
 
-store.dispatch(featureActions.detectSecurityLabellerFlags);
-store.dispatch(featureActions.detectCalicoFlags);
-store.dispatch(featureActions.detectOpenShift);
-store.dispatch(featureActions.detectCanListNS);
+
+_.forOwn(featureActions, (val, fnName) => {
+  store.dispatch(featureActions[fnName]);
+});
```

Comment:
Untested, but I'm wondering it this works

```js
_.each(featureActions, store.dispatch);
```

Reply:
Seems likely it would work, let me check.

Reply:
Yup, works. Nice.

Reply:
Now these could all just be anonymous functions & prob clean up a bunch here, but I won't worry about this now.  The named fns might be helpful for debugging anyway.